packages/compass/src/main/menu.ts

```diff
@@ -528,19 +443,15 @@ class CompassMenu {
       'window:hide-collection-submenu': this.hideCollection.bind(this),
     });
 
-    ipcMain.respondTo({
-      'window:theme-loaded': (_, theme) => {
-        if (this.themeState.theme !== theme) {
-          this.themeState.theme = theme;
-          this.saveThemeAndRefreshMenu({
-            theme
-          });
+    preferences.onPreferenceValueChanged('theme', newTheme => {
+      CompassTelemetry.track({
```

Comment:
Asking because I'm adding tracking in main process in another PR and I am doing this through the usual `createLoggerAndTelemetry`:

https://github.com/mongodb-js/compass/blob/b0cb4fc4940d55871ea7dffe261b22c7d580cc98/packages/compass/src/main/auto-update-manager.ts#L156

Should I use `CompassTelemetry` directly instead?

Reply:
No, actually, let me fix this while I’m here :+1: